radicle-httpd/src/api/v1/sessions.rs

```diff
@@ -82,13 +103,14 @@ async fn session_signin_handler(
             .checked_add(AUTHORIZED_SESSIONS_EXPIRATION)
             .unwrap();
         let session = Session {
+            status: AuthState::Authorized,
             public_key: request.pk,
-            issued_at: DateTime(OffsetDateTime::now_utc()),
+            issued_at: session.issued_at.to_owned(),
             expires_at: DateTime(expiration_time),
         };
-        sessions.insert(session_id.clone(), AuthState::Authorized(session));
+        sessions.insert(session_id.clone(), session);
```

Comment:
I wonder, could you do a `get_mut()` and then just mutate those fields, instead of having to recreate a session object and inserting?

Reply:
Yeah that would work, done in 1150daa4e04ef0195d3e478beb3840fcf5b014fe